File: Sesion.py
MIDDLEWARE = [
    # ...
    'django.contrib.sessions.middleware.SessionMiddleware',
    # ...
]

from django.http import HttpResponse
from django.shortcuts import render

def contar_visitas(request):
    # Intenta obtener el valor de 'visitas' de la sesión
    num_visitas = request.session.get('visitas', 0) + 1
    # Actualiza el valor de 'visitas' en la sesión
    request.session['visitas'] = num_visitas
    
    # Responde con el número de visitas
    return HttpResponse(f"Esta página ha sido visitada {num_visitas} veces.")

from django.urls import path
from . import views

urlpatterns = [
    path('visitas/', views.contar_visitas, name='contar_visitas'),
]

File: sesion.php
use Illuminate\Support\Facades\Route;
use App\Http\Controllers\VisitaController;

Route::get('/visitas', [VisitaController::class, 'contarVisitas']);

//codigo de crear controlador
php artisan make:controller VisitaController

namespace App\Http\Controllers;

use Illuminate\Http\Request;

class VisitaController extends Controller
{
    public function contarVisitas(Request $request)
    {
        // Incrementa el contador de visitas en la sesión.
        $visitas = $request->session()->increment('visitas', 1);

        // Devuelve una respuesta con el número de visitas.
        return "Esta página ha sido visitada {$visitas} veces.";
    }
}
